pkg/remote/builtin/github/github.go

```diff
@@ -34,12 +37,14 @@ type GitHub struct {
 
 func New(conf *config.Config) *GitHub {
```

Comment:
> and most of that code already exists in the 0.3 (master) branch. Just needs to be adapted to 0.4

cool!! :smile:  Can I add gitlab adapter to 0.4?
We want to really use GitLab (or Gogs).